pkg/memory-monitor/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb as memory-monitor-build
```

Comment:
```suggestion
FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 as memory-monitor-build
```

Reply:
Do we have a reliable way to get this hash?... And why do we use it at all?.. Why not semantic tags?

Reply:
You are hereby invited to join the discussion on https://github.com/lf-edge/eve/pull/3934 ;-)

Reply:
Fixed

Reply:
Fixed